roles/bind-keycloak-apb/templates/client_bearer.json.j2

```diff
@@ -1,7 +1,7 @@
 {
-  "id": "{{ CLIENT_ID }}",
-  "clientId": "{{ service }}-bearer",
-  "name": "{{ service }}",
+  "id": "{{ BEARER_CLIENT_ID }}",
+  "clientId": "{{ BEARER_CLIENT_NAME }}",
```

Comment:
Should not be `clientId`?

Reply:
clientId is the display name in keycloak, not related to the clientId we pass in as a param